eye2bids/edf2bids.py

```diff
@@ -221,15 +221,12 @@ def _extract_RecordedEye(df: pd.DataFrame) -> str:
 
 
 def _extract_ScreenResolution(df: pd.DataFrame) -> list[int]:
-    return list(
-        map(
-            int,
-            (
-                df[df[2] == "DISPLAY_COORDS"]
-                .iloc[0:1, 3:5]
-                .to_string(header=False, index=False)
-            ).split(" "),
-        )
+    return (
+        (df[df[2] == "GAZE_COORDS"])
+        .iloc[0:1, 3:5]
+        .to_string(header=False, index=False)
+        .replace(".00", "")
+        .split(" ")
```

Comment:
this should return integer values and currently it is returning strings

that's why the tests are failing